lerna.json

```diff
@@ -0,0 +1,8 @@
+{
+  "lerna": "2.11.0",
+  "packages": [
+    "packages/*"
```

Comment:
You can use `"packages/**"` to have packages such as `packages/inquirier/` and `packages/@inquirer/core/` if preferred